src/fs.ts

```diff
@@ -9,12 +9,12 @@ const writeFileAsync = util.promisify(fs.writeFile);
 const existsAsync = util.promisify(fs.exists);
 
 export async function exists(path: string, cwd = ''): Promise<boolean> {
-  return existsAsync(join(cwd, path));
+  return await existsAsync(join(cwd, path));
 }
 
 export async function readText(path: string, cwd = ''): Promise<string> {
   try {
-    return readFileAsync(join(cwd, path), { encoding: 'utf8' });
+    return await readFileAsync(join(cwd, path), { encoding: 'utf8' });
   } catch (e) {
     return '';
   }
```

Comment:
not `await`ing here caused and errors to reject the returned promise rather than landing in the `catch` block and returning the dummy value.